setup_python.sh

```diff
@@ -66,7 +66,6 @@ case $DD_TARGET_ARCH in
 
     python3 -m pip install distro==1.4.0 wheel==0.40.0
     python3 -m pip install --no-build-isolation "cython<3.0.0" PyYAML==5.4.1
-    python3 -m pip install -r requirements.txt
```

Comment:
Oversight, should have been part of this https://github.com/DataDog/datadog-agent-buildimages/pull/735